src/main/java/mate/jdbc/model/Car.java

```diff
@@ -0,0 +1,74 @@
+package mate.jdbc.model;
+
+import java.util.List;
+import java.util.Objects;
+
+public class Car {
+    private Long id;
+    private String model;
+    private Manufacturer manufacturer;
+    private List<Driver> drivers;
+
+    public Car() {
+    }
```

Comment:
if you removed the constructor with parameters, do you need to add the constructor without them explicitly?

Reply:
No, no need